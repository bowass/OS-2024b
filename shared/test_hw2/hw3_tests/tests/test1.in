open 0 0 read|write
ioctl 0 zero
ioctl 0 change_key 0
ioctl 0 change_read_state decrypt
write 0 "Hello World"
open 0 1 read|write
read 1 11
close 0
exit
