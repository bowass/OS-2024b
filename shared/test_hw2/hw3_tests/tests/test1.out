SUCCESS
SUCCESS
SUCCESS
SUCCESS
SUCCESS
SUCCESS
SUCCESS - Hello World
SUCCESS
